src/ddb/ddb_vos.h

```diff
@@ -142,6 +151,7 @@ struct dv_dtx_committed_entry {
 
 struct dv_dtx_active_entry {
 	uuid_t		ddtx_uuid;
+	uint64_t	ddtx_id_hlc;
```

Comment:
Why not directly use "struct dtx_id"?

Reply:
I was just flattening all the structs, but probably better to use dtx_id ... then can use DF/DP_DTI :-)